lua/entities/gmod_wire_expression2/core/cl_http_cl.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
```

Comment:
Like I said, let's move this to the coding style disussion and leave it as it is right now.
I was hoping @Divran would chip in an point to that discussion thread because I can't find it :)